src/backend/app/PaymentHandler/AccessRate.php

```diff
@@ -4,48 +4,46 @@
 
```

Comment:
Let's use "Device" instead of "Meter" in this whole class.

Reply:
I will move this to a separate refactor issue so as to keep this PR changes in scope. Thanks for the pointing out.

#530